<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doimatkhau</class>
 <widget class="QDialog" name="Doimatkhau">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>494</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mật khẩu cũ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Mật khẩu mới</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Nhập lại mật khẩu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMkcu">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linemkMoi">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>155</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMkmoix2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black</string>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
